modern-business/style-editor.scss

```
@@ -21,7 +21,7 @@ body {
 
 	@include media(mobile) {
 
-		.wp-block[data-align="full"] {
+		.wp-block[data-align="full"], .wp-block[data-type="a8c/post-content"] .wp-block[data-align="full"] {
```

According to the [WordPress CSS Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#structure), each selector should be in its own line.
```suggestion
		.wp-block[data-align="full"], 
		.wp-block[data-type="a8c/post-content"] .wp-block[data-align="full"] {
```

---

Thanks for the tip! I'll try to get some sort of plugin going in VS Code so I see any of those errors while I'm working. I'll also get those fixed in the meantime!